makers/appimage/src/cache.ts

```diff
@@ -0,0 +1,47 @@
+import envPaths from 'env-paths';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import * as url from 'url';
+import * as crypto from 'crypto';
+
+const defaultCacheRoot = envPaths('reforged', {
+  suffix: '',
+}).cache;
```

Comment:
Maybe just use:
```suggestion
import { homedir } from 'node:os';

const defaultCacheRoot = `${ process.env["XDG_CACHE_HOME"] ?? `${ homedir() }/.cache` }/reforged`;
```
…as maker currently runs on Linux only + I would like to avoid `envPaths` dependency.